app/pathogen/arbovirus/dashboard/page.tsx

```diff
@@ -5,18 +5,29 @@ import { VisualizationsSection } from "./(visualizations)/visualizations-section
 import { ArboDataTable } from "./(table)/ArboDataTable";
 import { ArbovirusMap } from "./(map)/ArbovirusMap";
 import { ArbovirusPageSectionId } from "../../../constants";
+import { useIsLargeScreen } from "@/hooks/useIsLargeScreen";
 
 export default function ArbovirusDashboardPage() {
+
+  const isLargeScreen = useIsLargeScreen();
+
   return (
-    <div className="col-span-12 row-span-2 grid gap-0 grid-cols-12 grid-rows-2 grid-flow-col w-screen overflow-hidden border-box">
-      <div className="overflow-y-scroll col-span-2 h-full row-span-2 border-y">
+    <div className="col-span-12 grid gap-0 grid-cols-12 grid-rows-4 lg:grid-rows-2 row-span-2 grid-flow-col w-screen overflow-hidden border-box">
+      <div className="overflow-y-scroll col-span-12 row-span-1 lg:col-span-2 lg:row-span-4 h-full border-y border-b border-background lg:border-b-0">
         <Filters
           className="p-4 border-background"
         />
       </div>
       <div
-        className={"overflow-y-scroll snap-y scroll-smooth col-span-10 row-span-2 px-4"}
+        className={"overflow-y-scroll snap-y scroll-smooth row-span-3 col-span-12 lg:col-span-10 lg:row-span-2 px-4"}
       >
+        {!isLargeScreen && (
+          <section
+          className="w-full h-fit relative row-span-2 rounded-md mt-4 border border-background p-4"
+        >
+          We suggest viewing our dashboard on a larger screen or in landscape mode for the best experience. Support for smaller devices is coming soon!
+        </section>
```

Comment:
I actually wonder if we could remove the whole `isLargeScreen` thing and just conditionally put `display: hidden` on this with a tailwind breakpoint for screen sizes under `lg` (The tailwind class is called `hidden` I think?). That way we're doing it all in css.

I feel like having stuff with `display: hidden` makes it not occupy space in the grid but I could be wrong?
Just an idea if you haven't given that a go already hahaha

Reply:
If your idea with the table was to make the column not sticky on certain screen sizes, I feel like that could probably be done in the css of the fixed column so I feel like that's NABD either.